backend/service/tasks/resources.py

```diff
@@ -4,15 +4,15 @@
 
 async def with_timeout(coroutine, timeout, **kwargs):
     try:
-        result = await asyncio.wait_for(coroutine(**kwargs), timeout=timeout)
-        return result
+        return asyncio.wait_for(coroutine(**kwargs), timeout=timeout)
     except asyncio.TimeoutError:
         return -1  # defined timeout as -1
 
 
 def task(task_name: str):
     def wrapper(func):
         async def async_wrapper(**kwargs):
-            return with_timeout(coroutine=func, timeout=MAX_TIMEOUT_SECONDS, **kwargs)
+            return func(**kwargs)
+            # return await with_timeout(coroutine=func, timeout=MAX_TIMEOUT_SECONDS, **kwargs)
```

Comment:
I had to remove the timeout functionality since was not working. I got the folling error:
` /app/code/service/workers/worker.py:57: RuntimeWarning: coroutine 'wait_5s' was never awaited
 `